src/openloco/localisation/stringmgr.cpp

```diff
@@ -1,23 +1,649 @@
+                            unit = "ft";
+                        }
+                        else
+                        {
+                            // !!! TODO: Move to string id
+                            unit = "m";
+                            value = (value * 840) >> 8;
+                        }
+
+                        buffer = format_comma(value, buffer);
+
+                        do
+                        {
+                            *buffer = *unit;
+                            buffer++;
+                            unit++;
+                        } while (*unit != '\0');
+
+                        *buffer = '\0';
+
+                        break;
+                    }
+
+                    case 123 + 18:
+                    {
+                        // height
+                        uint32_t value = args.pop16();
+
+                        bool show_height_as_units = config::get().flags & config::flags::SHOW_HEIGHT_AS_UNITS;
+                        uint8_t measurement_format = config::get().measurement_format;
+                        const char* unit;
+
+                        if (show_height_as_units)
+                        {
+                            // !!! TODO: move to string id
+                            unit = " units";
+                        }
+                        else if (measurement_format == config::measurement_formats::FORMAT_IMPERIAL)
+                        {
+                            // !!! TODO: Move to string id
+                            unit = "ft";
+                            value *= 10;
+                        }
+                        else
+                        {
+                            // !!! TODO: Move to string id
+                            unit = "m";
+                            value *= 5;
+                        }
+
+                        buffer = format_comma(value, buffer);
+
+                        do
+                        {
+                            *buffer = *unit;
+                            buffer++;
+                            unit++;
+                        } while (*unit != '\0');
+
+                        *buffer = '\0';
+
+                        break;
+                    }
+
+                    case 123 + 19:
+                    {
+                        // power
+                        uint32_t value = args.pop16();
+                        auto measurement_format = config::get().measurement_format;
+
+                        const char* unit;
+                        if (measurement_format == config::measurement_formats::FORMAT_IMPERIAL)
+                        {
+                            // !!! TODO: Move to string id
+                            unit = "hp";
+                        }
+                        else
+                        {
+                            // !!! TODO: Move to string id
+                            unit = "kW";
+                            value = (value * 764) >> 10;
```

Comment:
I think multiplying by `0.74609375` would be more obvious; and since it's just visual, I don't think the "integer math only" rule is that important.